Leanplum-SDK/Classes/Leanplum.h

```diff
@@ -256,6 +256,16 @@ NS_SWIFT_NAME(setAppId(_:developmentKey:));
 NS_SWIFT_NAME(setAppId(_:productionKey:));
 /**@}*/
 
+/**
+ * Set the appId and Development key from the plist.
+*/
++ (void)setAppForDevelopmentUsingPlist;
```

Comment:
Those methods shouldn't be exposed to client, instead `Leanplum.m` `+(void)load` should try to load plist file from main bundle and set the keys.

Reply:
In this case, should we use the DEBUG flag to decide between Prod and Dev key usage? 
It seems this way it will not be possible to use the Prod key while testing with debug build, which is a common use case.